.gitignore

```diff
@@ -13,4 +13,10 @@ Gemfile.lock
 beautiful-jekyll-theme-*.gem
 
 # PyCharm
-/.idea
\ No newline at end of file
+/.idea
+
+# Python stuff
+__pycache__/
+
+# Lock files
+.~lock.*.csv#
```

Comment:
Please use local exclusions (`.git/info/exclude`) for files that are specific to your preferred apps (and not the project in general).